### DISCUSSION TITLE
gcp/2019/07/20/bigquery-with-spreadsheet/

### DISCUSSION THREAD

**Discussion by zzsza** [zzsza/blog-comments#113] [Comments]
#  Google Spreadsheets(Sheets)에서 BigQuery 사용하기 

https://zzsza.github.io/gcp/2019/07/20/bigquery-with-spreadsheet/

**Comment by zzsza** [comment]
<p>OWOX 사용방법에 <a href="http://sheet.new" rel="nofollow noopener" title="sheet.new">sheet.new</a> 링크가 연결아 안되는것 같습니다!! ㅜ</p>

>_This comment was made by Disqus. 2019-09-16 11:25:11 박이삭_

**Comment by zzsza** [comment]
<p>OWOX를 사용하기 위해서는 빅쿼리를 사용하는 계정으로 스프레드 시트를 열어야 하는 건가요?</p>

>_This comment was made by Disqus. 2019-09-16 11:36:24 박이삭_

**Comment by gazi-kichan** [comment]
감사합니다! 포스팅 도움받아 OWOX로 빅쿼리와 시트 데이터 연동에 성공했습니다 :)

관련해서 궁금한 것이 있어 질문 남깁니다. 

매일0시 기준 전날 데이터를 불러오는데 기존 데이터는 그대로두고 기존 데이터 마지막 행에서 다음 데이터를 계속 불러오게 설정 할 수 있을까요?
